src/main/java/frc/robot/commands/DriveManual.java

```diff
@@ -46,7 +46,7 @@ public void execute() {
     ChassisSpeeds velocities = subDrivetrain.calculateVelocitiesFromInput(
         xAxis,
         yAxis,
-        rotationAxis,
+        () -> -rotationAxis.getAsDouble(),
```

Comment:
I think this wrong. By doing it here you are just inverting for manual. what about PoseDrive? I think we would want to change the isSimulation thing in ConstDriveteam